.devops/deploy-pipelines.yml

```
@@ -329,3 +329,72 @@ stages:
                     VALUE_FILE: "helm/pay-wallet-values-uat.yaml"
                     GREEN_VERSION: $(app_version)
   # --- END Deploy UAT --- #
+
+  # --- START of Validate event in Hub (UAT) --- #
+
+  - stage: "UAT_Testing_Generate_Wallet_Event"
+    displayName: "UAT Testing Generate Wallet Event"
+    dependsOn: [ Deploy_UAT_Green]
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - job: ValidateCDCFlow
+        displayName: 'Run API Tests DataLake Flow (UAT)'
+        steps:
+          - script: |
+              yarn global add newman
+            displayName: 'newman installation'
+
+          - script: |
+              newman run --ignore-redirects api-tests/wallet_payment_api.cards.tests.json --environment=api-tests/uat.cards.envs.json --env-var WALLET_TOKEN_TEST=$(WALLET_TOKEN_TEST) --reporters cli,junit,json > output.txt
+            displayName: 'Run API tests (UAT)'
+
+          - script: |
+              # Extract WALLET_ID from report JSON
+              WALLET_ID=$(grep -oP "WALLET_ID=\K[^\']+" output.txt)
+              
+              if [ -z "$WALLET_ID" ]; then
+                echo "WALLET_ID non trovato nel report."
```

```suggestion
                echo "Cannot extract WALLET_ID from report"
```

please use english for messages